### PR TITLE
Feat/check literal values

### PR DESCRIPTION
**Pull request by thepassle**
fixes https://github.com/open-wc/open-wc/issues/1913

### CODE REVIEW

**Comment by thepassle** [comment]
@bennypowers any idea how to fix this? 
image

File: packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js [hunk 1]
```diff
@@ -50,21 +51,23 @@ const AriaRoleRule = {
                   return;
                 }
 
-                if (rawValue.startsWith('{{')) {
+                const val = getExpressionValue(analyzer, rawValue);
+
+                if (!val && rawValue.startsWith('{{')) {
```
Comment (bennypowers) [inline packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js:56]: maybe a helper `isInterpolation`?

File: packages/eslint-plugin-lit-a11y/lib/rules/img-redundant-alt.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ const ImgRedundantAltRule = {
     },
     messages: {
       imgRedundantAlt:
-        '<img> alt attribute must be descriptive; it cannot contain the banned {{plural}} {{formatted}}.',
+        '<img> alt attribute must be descriptive; it cannot contain the banned {{plural}} {{banned}}.',
```
Comment (bennypowers) [inline packages/eslint-plugin-lit-a11y/lib/rules/img-redundant-alt.js:42]: Good catch!

**Review by bennypowers — APPROVED** [review]

File: packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js (hunk 1 above)
Comment (thepassle) [reply packages/eslint-plugin-lit-a11y/lib/rules/aria-role.js:56]: yeah we can do, id prefer the name `isPlaceholder` though

**Pull request merged**